test/vendor/github.com/Microsoft/hcsshim/ext4/tar2ext4/tar2ext4.go

```diff
@@ -268,3 +263,37 @@ func ReadExt4SuperBlock(vhdPath string) (*format.SuperBlock, error) {
 	}
 	return &sb, nil
 }
+
+// ConvertAndComputeRootDigest writes a compact ext4 file system image that contains the files in the
+// input tar stream, computes and returns its cryptographic digest. Convert is called with minimal
+// options: ConvertWhiteout and MaximumDiskSize set to dmverity.RecommendedVHDSizeGB.
+func ConvertAndComputeRootDigest(r io.Reader) (string, error) {
```

Comment:
Comment implies this computes the digest of the converted stream but it actually returns root digest of the computed Merkel tree.

Reply:
yeah, I was hoping that `cryptographic digest` is a good enough term, but I agree that it's better to spell things out.

Reply:
done